checker/src/main/java/org/checkerframework/checker/calledmethods/CalledMethodsTransfer.java

```diff
@@ -61,6 +62,16 @@ public CalledMethodsTransfer(final CalledMethodsAnalysis analysis) {
         ((CalledMethodsAnnotatedTypeFactory) atypeFactory).calledMethodsValueElement;
   }
 
+  @Override
+  protected boolean shouldPerformWholeProgramInference(Tree tree) {
+    return false;
```

Comment:
These should definitely be controlled by whether or not the switch for RLC inference is enabled, rather than `false` all the time. I'm not sure exactly what the intended specification is, though, because there isn't documentation for how RLC inference interacts with WPI; I think there should be, in the Checker Framework manual somewhere.